packages/grafana-data/src/types/variables.ts

```diff
@@ -0,0 +1,17 @@
+import { OperatorFunction } from 'rxjs';
+
+import { DataQuery, DataQueryRequest, MetricFindValue } from './datasource';
+import { DataFrame } from './dataFrame';
+import { ScopedVars } from './ScopedVars';
+
+export interface VariableSupport<TQuery extends DataQuery = DataQuery> {
+  /**
+   * Defines how to transform the query provided from the variable into the DataQueryRequest used to query for MetricFindValues.
+   */
+  toDataQueryRequest: (query: string | any, scopedVars: ScopedVars) => DataQueryRequest;
```

Comment:
Think `any` just overrides `string` here, so it might as well just be `any`.

Reply:
I know, I just wanted to communicate that for 99% of all current core plugins this is a string but then there is just one that passes an object.

Reply:
I changed it to `any` makes it very clear that you should expect anything :) Thank you Kay!